utils/das_data_sorter.py

```diff
@@ -136,6 +137,53 @@ def gps_data(self, data:pd.Series) -> pd.Series:
         
         return new_gps_data
     
+    def smooths_data(self, data:pd.Series, technique:str, n:int=3) -> pd.Series:
+        if n < 3 or n > len(data):
+            raise ValueError("Number of smoothing points must be at least 3 and less than the length of the data set to perform smoothing.")
+        smooth_data_array = [] 
+
+        if technique == "mean" and n % 2 != 0: # Mean Smoothing for odd number N
+            for i in range(len(data) - N + 1):
```

Comment:
Where is `N` defined? Also, in line with `snake_case` conventions, it's probably better to call the variable `n`.

Reply:
Oh whoops sorry, that's a typo. It is meant to be lowercase n